server/meshmodel/argo-cd/relationships/hierarchical_inventory.json

```diff
@@ -0,0 +1,77 @@
+{
+  "apiVersion": "core.meshery.io/v1alpha1",
+  "kind": "Hierarchical",
+  "metadata": {
+    "description": "A hierarchical inventory relationship in which the configuration of (parent) component is patched with the configuration of other (child) component. Eg: The configuration of the EnvoyFilter (parent) component is patched with the configuration as received from WASMFilter (child) component."
```

Comment:
I wonder if we have a more readily comprehensible example for the Kubernetes layman as opposed to the somewhat advanced and niche EnvoyFilter example. A quick review of the rest of the existing policies might offer an alternative example such as Pod (child) and Namespace (parent).

Reply:
Hierarchical-Parent makes sense for this case if we can relate it to namespaces and pods. Project can be related to namespace.

Reply:
Great. Given this, will you please update the sample text with a more readily comprehensible example for users?